mathesar_ui/src/components/NameWithIcon.svelte

```diff
@@ -9,12 +9,13 @@
   /** When true, the icon will be rendered within a box */
   export let iconHasBox = false;
   export let truncate = true;
+  export let bold = false;
```

Comment:
This seems a bit concerning for the future. We might end up having a lot of booleans for varying CSS props. No changes are needed urgently though.